jukebox/hparams.py

```diff
@@ -217,6 +217,44 @@ def setup_hparams(hparam_set_names, kwargs):
 )
 HPARAMS_REGISTRY["small_prior"] = small_prior
 
+small_single_enc_dec_prior = Hyperparams(
+    n_ctx=6144,
+    prior_width=1024,
+    prior_depth=48,
+    heads=4,
```

Comment:
nitpick: in general, lgtm, but perhaps, 2 heads will be enough? 1B lyrics uses 2 heads as well

Reply:
good point, changing!